src/Components/test/E2ETest/Infrastructure/WebDriverExtensions/BasicTestAppAuthenticationWebDriverExtensions.cs

```diff
@@ -4,7 +4,7 @@
 
 namespace Microsoft.AspNetCore.Components.E2ETest
 {
-    internal static class AuthHelpers
+    internal static class BasicTestAppAuthenticationWebDriverExtensions
```

Comment:
~~Helpers~~

I'm sure @rynowak will be pleased :)

Reply:
Yeah, git is doing weird stuff. This class/file is new